packages/manager/modules/hub/src/components/user-panel/user-infos/template.html

```diff
@@ -0,0 +1,69 @@
+<div class="manager-hub-user-infos">
+    <span
+        class="initials"
```

Comment:
Maybe this could be prefix to have a better scope ? (what if bootstrap provides a utility with the same selector ? )